packages/plugins/typescript-resolvers/src/root.handlebars

```diff
@@ -65,4 +65,25 @@ export type DirectiveResolverFn<TResult, TArgs = {}, TContext = {}> = (
 
 {{#each scalars}}
 {{~> scalar}}
-{{/each}}
\ No newline at end of file
+{{/each}}
+
+export interface IResolvers {
+  {{#each types}}
+    {{ convert name }}?: {{ convert name }}Resolvers{{#unless @root.config.noNamespaces}}.{{/unless}}Resolvers;
+  {{/each}}
+  {{#each interfaces}}
+    {{ convert name }}?: {{ convert name }}Resolvers{{#unless @root.config.noNamespaces}}.{{/unless}}Resolvers;
+  {{/each}}
+  {{#each unions}}
+    {{ convert name }}?: {{ convert name }}Resolvers{{#unless @root.config.noNamespaces}}.{{/unless}}Resolvers;
+  {{/each}}
+  {{#each scalars}}
+    {{ convert name }}?: GraphQLScalarType;
```

Comment:
@ardatan scalars have now their own resolvers too, maybe we could update that line, what do you think?

Reply:
We have ScalarTypeConfig interface, but I couldn't find out the way to use it here, because resolvers take ScalarType which doesn't have any generic.

Reply:
@ardatan I did some work for scalars recently: https://github.com/dotansimha/graphql-code-generator/blob/master/packages/plugins/typescript-resolvers/src/scalar.handlebars